bigframes/core/ordering.py

```diff
@@ -215,6 +215,15 @@ def is_sequential(self) -> bool:
     def all_ordering_columns(self) -> Sequence[OrderingExpression]:
         return list(self.ordering_value_columns)
 
+    @property
+    def referenced_columns(self) -> Sequence[str]:
+        cols = []
+        for part in self.ordering_value_columns:
+            for col in part.scalar_expression.unbound_variables:
+                if col not in cols:
```

Comment:
Does the order of `cols` matter? If not a `set()` may be more appropriate than a list.

Reply:
Changed to set, caller can sort for determinism